release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1289,6 +1289,40 @@ Any registry on this list is not required to have an entry in the pull secret us
 
+[id="ocp-4-12-0-ironic-agent-image-ztp"]
+==== Ironic agent mirroring in disconnected GitOps ZTP installations
+
+For disconnected installations using GitOps ZTP, if you are deploying {product-title} version 4.11 or earlier to a spoke cluster with converged flow enabled, you must mirror the default Ironic agent image to the local image repository. The default Ironic agent images are the following:
+
+* AMD64 Ironic agent image: `quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:d3f1d4d3cd5fbcf1b9249dd71d01be4b901d337fdc5f8f66569eb71df4d9d446`
+
+* AArch64 Ironic agent image: `quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:cb0edf19fffc17f542a7efae76939b1e9757dc75782d4727fb0aa77ed5809b43`
+
+For more information about mirroring images, see xref:../installing/disconnected_install/installing-mirroring-installation-images.adoc#installation-mirror-repository_installing-mirroring-installation-images[Mirroring the OpenShift Container Platform image repository]
+
+[id="support-for-dual-stack-addressing-for-the-API-VIP-and-Ingress-VIP"]
+= Support for dual-stack addressing for the API VIP and Ingress VIP
+{ai-full} supports installation of {product-title} 4.12 and later versions with dual stack networking for the API VIP and Ingress VIP on bare metal only. This support introduces two new configuration settings: `api_vips` and `ingress_vips`, which can take a list of IP addresses. The legacy settings, `api_vip` and `ingress_vip` must also be set in {product-title} 4.12; however, since they only take one IP address, you must set the IPv4 address when configuring dual stack networking for the API VIP and Ingress VIP with the legacy `api_vip` and `ingress_vip` configuration settings.
+
+The API VIP address and the Ingress VIP address must be of the primary IP address family when using dual-stack networking. Currently, Red Hat does not support dual-stack VIPs or dual-stack networking with IPv6 as the primary IP address family. However, Red Hat does support dual-stack networking with IPv4 as the primary IP address family. Therefore, you must place the IPv4 entries before the IPv6 entries. See the {ai-full} documentation for details.
+
+[id="post-installation-configuration"]
+= Assisted Installer SaaS provides platform integration support for Nutanix
+{ai-full} SaaS on link:https://console.redhat.com[console.redhat.com] supports installation of {product-title} on the Nutanix platform with Machine API integration using either the {ai-full} user interface or the REST API. Integration enables Nutanix Prism users to manage their infrastructure from a single interface, and enables auto-scaling. There are a few additional installation steps to enable Nutanix integration with {ai-full} SaaS. See the {ai-full} documentation for details.
```

Comment:
Hi @racedo, I think if a release note is needed for MGMT-13374, it should go into the ACM release notes, not the OpenShift release notes. I'm also hesitant about putting AI SaaS release notes in the OpenShift docs since AI SaaS has it's own way of announcing features via the What's New.